docs/usage.md

```diff
@@ -258,6 +258,48 @@ const registry = new SchemaRegistry({
 })
 ```
 
+HTTP Agent configuration offer a high degree of customization for underlying both HTTP and HTTPS requests. 
+
+#### TLS/SSL Authentication
+
+If your Schema Registry requires TLS/SSL Authentication you can pass a custom `https.Agent` to its constructor which accept the options available in [tls.createSecureSocket()](https://nodejs.org/docs/latest/api/tls.html#tlscreatesecurecontextoptions).
+
+```js
+import { Agent } from 'https'
+import * as fs from 'fs'
+
+const agent = new Agent({ 
+    ca: [ fs.readFileSync('/path/to/yourca.crt', 'utf-8') ],
```

Comment:
Seems unnecessary to wrap in `[]` because it accepts both `string` or `string[]`.

Reply:
I can confirm it works just as well without the `[]`.